pennylane/qnodes/base.py

```diff
@@ -386,6 +386,8 @@ def _construct(self, args, kwargs):
         # generate the DAG
         self.circuit = CircuitGraph(self.ops, self.variable_deps)
 
+        self.circuit_hash = self.circuit.circuit_hash
```

Comment:
This line can probably be removed, since the hash is accessible anyway via `qnode.circuit.hash`

Reply:
That's true, thanks for that!